app.js

```diff
@@ -1,31 +1,31 @@
-// const express = require('express');
-// const routes = require("./routes")
-// const fs = require("fs");
+const express = require('express');
+const routes = require("./routes")
+const fs = require("fs");
 
-// const app = express();
-// const port = 3000;
+const app = express();
```

Comment:
Trying to understand if the sequence is correct here.
Does the routes getting assigned to the app even when they are imported/initialized before the app was initialized?

Reply:
Alright, line two `const routes = require("./routes");` does the magic - it is the import of the route functions not libraries.

Reply:
Done

Reply:
I actually thought this twice so I let only a get('/') respond so if by any chance you open the browser at the servers port , you can have a nicely format msg!